aladdin.sh

```diff
@@ -52,6 +62,15 @@ function get_plugin_dir() {
     fi
 }
 
+function get_manage_minikube() {
```

Comment:
now getting MANAGE_MINIKUBE value from config.
Must be set like this: `aladdin config set manage_minikube false`
`true` is the default value because that's how aladdin works today.